src/main/webapp/WEB-INF/tags/page.tag

```diff
@@ -13,11 +13,12 @@
     <link rel="shortcut icon" href="/favicon.png" />
 
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0">
```

Comment:
Yup, zooming screws up the header and the footer (which are vanilla implementations rather than the bootstrap versions for some reason). Users should not require zooming on this responsive version.